clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java

```diff
@@ -8,6 +8,7 @@
 import com.linkedin.avroutil1.compatibility.AvroCompatibilityHelper;
 import com.linkedin.avroutil1.compatibility.AvroVersion;
 import com.linkedin.venice.HttpConstants;
+import com.linkedin.venice.VeniceConstants;
```

Comment:
**Suggestion:** Add missing import statement for VeniceConstants
```suggestion
+import com.linkedin.venice.VeniceConstants;

```

Reply:
@CodiumAI-Agent /review